WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/AddProductToOrderViewModel.swift

```diff
@@ -139,6 +162,29 @@ extension AddProductToOrderViewModel: SyncingCoordinatorDelegate {
         stores.dispatch(action)
     }
 
+    /// Sync products matching a given keyword.
+    ///
+    private func searchProducts(siteID: Int64, keyword: String, pageNumber: Int, pageSize: Int, onCompletion: ((Bool) -> Void)?) {
+        let action = ProductAction.searchProducts(siteID: siteID,
+                                                  keyword: keyword,
+                                                  pageNumber: pageNumber,
+                                                  pageSize: pageSize) { [weak self] result in
+            guard let self = self else { return }
+
+            switch result {
+            case .success:
+                self.updateProductsResultsController()
+            case .failure(let error):
+                DDLogError("⛔️ Error searching products during order creation: \(error)")
+            }
```

Comment:
Should we be notifying these errors?

Reply:
Yes! I thought about adding error notices here but there are other places in the Products section we should also add them, and I didn't want to do too many things in one PR. I've tracked that in https://github.com/woocommerce/woocommerce-ios/issues/6119 so we can follow up with those notices.